### PR TITLE
docs: update README for 2U fork

### PR DESCRIPTION
**Pull request by robrap**
----

I've completed each of the following or determined they are not applicable:

- [ ] Made a plan to communicate any major developer interface changes (or N/A)

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,60 +1,55 @@
 Devstack
-#################
+########
 
 |ci-provisioning-badge| |ci-cli-badge| |doc-badge| |license-badge|
```
Comment (robrap) [inline README.rst]:
```suggestion
|ci-provisioning-badge| |ci-cli-badge| |license-badge|
```

File: README.rst [hunk 2]
```diff
@@ -1,60 +1,55 @@
...
+Overview
+********
 
-Going forward, devstack will be primarily used for development by 2U. To do development
-on Open edX, it is recommended that `Tutor`_ be used instead.
+This is 2U's version of devstack used for developing using the Open edX platform.
 
-For more information on this deprecation, please visit the `associated deprecation ticket`_.
+The Open edX version of devstack was deprecated and archived. Be careful to
+avoid any outdated documentation from the Open edX Read the Docs or the
```
Comment (robrap) [inline README.rst]:
```suggestion
avoid any outdated documentation from the Open edX Read the Docs or the
```

File: README.rst [hunk 3]
```diff
@@ -1,60 +1,55 @@
...
 
 Getting Help
 ************
 
 Documentation
 =============
 
-Start by going through `the documentation`_ on Read the Docs.  If you need more help see below.
+Start by going through `the documentation`_ on GitHub. Our documentation is not yet published
+to Read the Docs, so you should avoid the outdated Open edX docs on Read the Docs.
```
Comment (robrap) [inline README.rst]:
```suggestion
to Read the Docs, so you should avoid the outdated Open edX docs on Read the Docs.
```

**Review by robrap — COMMENTED** [review]
Fixing linting issues.

File: README.rst [hunk 4]
```diff
@@ -1,60 +1,55 @@
...
 
 
 Getting Started
 ***************
 
-The `Getting Started guide`_ lives with the rest of the documentation in Read the Docs.
+The `Getting Started guide`_ lives with the rest of the documentation.
 
-.. _Getting Started guide: https://edx.readthedocs.io/projects/open-edx-devstack/en/latest/getting_started.html
+.. _Getting Started guide: https://github.com/edx/devstack/blob/master/docs/getting_started.rst
```
Comment (timmc-edx) [inline README.rst:26]: What's the reason for switching these URLs from the rendered docs to the source? Do we no longer have the ability to push to edx.readthedocs.io?

Comment (robrap) [reply README.rst:26]: Yes. That is for the Open edX version, which they don't plan on retiring, but they are going to put up a banner that points to this file. We could publish to read the docs at some point again, but that is future work that I am not taking on, so this README update just reflects the current reality.

Comment (robrap) [reply README.rst:26]: Here is the ticket for publishing to Read the Docs again: https://github.com/edx/edx-arch-experiments/issues/578

Comment (timmc-edx) [reply README.rst:26]: I see, thanks!

**Review by timmc-edx — APPROVED** [review]
Looks good, just may want to call out the RTD issue directly in the commit message.

**Pull request merged**